### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by zivke**
1) Added a CHANGELOG.md file that covers all of the changes (by version) from the original IgH
2) Added Synapticon-specific stuff to the documentation (README and INSTALL)
3) Fixed the "sncn_installer/install.sh" (so that it completely installs the ECAT master onto a fresh Ubuntu system - tested)

I did not rename the "sncn_installer" directory because it is being used in several projects. Also, I tried to stay true to the original IgH installer, so that we don't have to change any of the original IgH instructions.

Once these changes are merged into develop, we will merge everything to master and create a new release (v1.5.2-sncn-7).

Any suggestions are welcome.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# Changelog
+All notable changes to this project made by SYNAPTICON will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+##[Unreleased]
+### EtherCAT wrapper library
```
Comment (jbck) [inline CHANGELOG.md]: No no no, please don't mention the wrapper here. You can make a CHANGELOG in the wrapper folder, but eventually the whole wrapper needs to be moved.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+- Fix: sncn_installer uses right sysconfig
+
+## [1.5.2-sncn-6] - 2018-10-23
+### EtherCAT wrapper library (v0.6)
+#### Added
+- Add version for libethercat_wrapper to distribution
+- Use syslog for logging
+- Add the object name to the SDO structure
+- Enable fetching some slave information before EtherCAT master init
+- Enable the wrapper to update slave states (ecw_master_scan)
```
Comment (jbck) [inline CHANGELOG.md]:
```suggestion
- Allow an update of the slave states (ecw_master_scan)
```
(or something like that; the point is, no wrapper mention)

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+- Add a function to trigger bus rescanning
+
+#### Fixed
+- Suport Linux kernel 4.15.7
+- Fix: sncn_installer uses right sysconfig
+
+## [1.5.2-sncn-6] - 2018-10-23
+### EtherCAT wrapper library (v0.6)
+#### Added
+- Add version for libethercat_wrapper to distribution
```
Comment (jbck) [inline CHANGELOG.md]: What was this? The ability get the version? In that case, just say "Add get_version()" (or whatever). Please no mention of the wrapper.

**Comment by zivke** [comment]
> This is great! But the wrapper stuff needs to be split apart.

I just meant to note all of the changes made to the wrapper in the changelog so we can extract it later more easily. But yeah, I can move the changelog into the wrapper directory, that also works. Thanks.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# Changelog
+All notable changes to this project made by SYNAPTICON will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+##[Unreleased]
```
Comment (jbck) [inline CHANGELOG.md]: Ahhhh dang it, I missed this. There needs to be a space after the hash to make it a heading.

File: libethercat_wrapper/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
+# Changelog
+All notable changes to this project made by SYNAPTICON will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+##[Unreleased]
```
Comment (jbck) [inline libethercat_wrapper/CHANGELOG.md]: Same here.

**Pull request merged**